### PR TITLE
Port staking changes

### PR DESCRIPTION
**Pull request by hyd628**
### Description/Original PRs

https://github.com/PureStake/moonbeam-docs/pull/222
https://github.com/PureStake/moonbeam-docs/pull/229

### Checklist

- [x] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [x] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [x] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

Please link to any additional corresponding PRs here.

### CODE REVIEW

File: node-operators/networks/collator.md [hunk 1]
```diff
@@ -11,49 +11,55 @@ description: 学习如何在运行节点后成为Moonbeam网络的收集人
...
-## 技术要求 {: #technical-requirements }
+## 技术要求 {: #technical-requirements } 
 
 从技术角度来看，收集人必须满足以下技术要求：
 
  - 必须运行带有收集选项的全节点。可根据[启动全节点教程](/node-operators/networks/run-a-node/overview/)选择收集人的特定代码段
 
-## 账户与质押要求 {: #accounts-and-staking-requirements }
+!!! 注意事项
+
```
Comment (eshaben) [inline node-operators/networks/collator.md:30]: I think this space here is making it so the note doesn't appear correctly

File: node-operators/networks/collator.md [hunk 2]
```diff
@@ -11,49 +11,55 @@ description: 学习如何在运行节点后成为Moonbeam网络的收集人
...
+​    运行一个**收集人**节点对CPU的要求高于上述所提供的要求。为了使您的收集人节点能够跟上高交易吞吐量，具有高时钟速度和单核性能的CPU非常重要，因为区块生产/导入过程几乎完全是单线程的。我们使用以下CPU进行了测试并看到了良好的结果：Intel Xeon E-2386/2388和Ryzen 9 5950x/5900x。由于大多数云服务提供商专注于多线程而非单线程性能，因此建议使用裸机提供商。另外，因为Docker将对性能产生重大影响，因此也不建议在Docker中运行收集人节点。
+
+## 账户与质押要求 {: #and-staking-requirements }
```
Comment (eshaben) [inline node-operators/networks/collator.md:33]: this should be `## Accounts and Staking Requirements {: #accounts-and-staking-requirements }`

File: node-operators/networks/collator.md [hunk 3]
```diff
@@ -351,45 +344,43 @@ curl http://127.0.0.1:9933 -H \
...
- 4. 选择**mappingWithDeposit**函数
+  4. 选择**mappingWithDeposit**函数
 
- 5. 提供author ID进行查询。您也可以通过关闭按钮以停止检索所有链上的映射情况
+  5. 提供author ID进行查询。您也可以通过关闭按钮以停止检索所有链上的映射情况
 
- 6. 点击**+**按钮来传送RPC调用
+   6. 点击**+**按钮来传送RPC调用
 
 ![Author ID Mapping Chain State](/images/node-operators/networks/collators/collator-polkadotjs-6.png)
```
Comment (eshaben) [inline node-operators/networks/collator.md:385]: After this there should be a section on Join the Discord: https://github.com/PureStake/moonbeam-docs/pull/229/files#diff-4a1e759dec4fe268e80b04913929273d86c98ee78cb33411197d028e1b3fb5b0R345-R355

File: learn/features/staking.md [hunk 4]
```diff
@@ -15,58 +15,123 @@ Moonbeam采用基于[波卡的权益证明（PoS）模型](https://wiki.polkadot
...
+    |       奖励发放延迟        |    {{ networks.moonbase.delegator_timings.rewards_payouts.rounds }}轮次 ({{ networks.moonbase.delegator_timings.rewards_payouts.hours }}小时)    |
+    |     增加委托时长    |                                           委托将会在下一个轮次生效（资金可随时提取）                                            |
+    |    减少委托时长     |      {{ networks.moonbase.delegator_timings.del_bond_less.rounds }}轮次 ({{ networks.moonbase.delegator_timings.del_bond_less.hours }}小时)      |
+    |     取消委托延迟     | {{ networks.moonbase.delegator_timings.revoke_delegations.rounds }}轮次 ({{ networks.moonbase.delegator_timings.revoke_delegations.hours }}小时) |
+    |      退出委托人集延迟      |   {{ networks.moonbase.delegator_timings.leave_delegators.rounds }}轮次 ({{ networks.moonbase.delegator_timings.leave_delegators.hours }}小时)   |
+
 
 想要获取任何质押参数的当前值，请参考[如何质押您的Token](/tokens/staking/stake/)教程的[检索质押参数](/tokens/staking/stake/#retrieving-staking-parameters)部分。
 
+If you're looking for candidate or collator-specific requirements and information, you can take a look at the [Collators](/node-operators/networks/collator) guide.
```
Comment (eshaben) [inline learn/features/staking.md:68]: This needs to be translated still

File: node-operators/networks/collator.md [hunk 5]
```diff
@@ -96,75 +112,62 @@ console.log(`Candidate pool size is: ${candidatePool.length}`);
...
-## 停止参与收集活动 {: #stop-collating }
+## 停止参与收集活动 {: #stop-collating } 
 
 在最新的runtime升级（[runtime version 1001](https://moonbeam.network/announcements/staking-changes-moonriver-runtime-upgrade/)）中，用户与质押功能的交互方式进行了重大升级，其中包含取消质押的方式。
 
-想要停止参与收集活动并离开候选人池，您必须先发起请求。发起请求并不会自动将您从候选人池中移除，您需要等待延迟时段，才能够执行请求并停止参与收集活动。在Moonriver上会有{{ networks.moonriver.collator_timings.leave_candidates.rounds }}轮（{{ networks.moonriver.collator_timings.leave_candidates.hours }}小时）的延迟。在Moonbase Alpha上会有{{ networks.moonbase.collator_timings.leave_candidates.rounds }}（{{ networks.moonbase.collator_timings.leave_candidates.hours }}小时）的延迟。在等待具体的轮次数的过程中，如果您在有效集中您仍有资格生产区块和获取奖励。
+想要停止参与收集活动并离开候选人池，您必须先发起请求。发起请求并不会自动将您从候选人池中移除，您需要等待[退出生效期](#collator-timings)，才能够执行请求并停止参与收集活动。在等待具体轮次数的过程中，如果您在有效集中您仍有资格生产区块和获取奖励。
 
-#### 发起离开候选人的请求 {: #schedule-request-to-leave-candidates }
+### 发起离开候选人的请求
```
Comment (eshaben) [inline node-operators/networks/collator.md:149]: this is missing a header attribute now, it should still be `{: #schedule-request-to-leave-candidates }`

File: node-operators/networks/collator.md [hunk 6]
```diff
@@ -96,75 +112,62 @@ console.log(`Candidate pool size is: ${candidatePool.length}`);
...
- 6. 输入`candidateCount`（可从[获取候选人池的大小](#get-the-size-of-the-candidate-pool)部分获得）
+  6. 输入`candidateCount`（可从[获取候选人池的大小](#get-the-size-of-the-candidate-pool)部分获得）
 
- 7. 提交交易。根据向导指引使用创建账户时的密码进行交易签名
+   7. 提交交易。根据向导指引使用创建账户时的密码进行交易签名
 
 ![Schedule leave candidates request](/images/node-operators/networks/collators/collator-polkadotjs-9.png)
 
-#### 执行离开候选人的请求 {: #execute-request-to-leave-candidates }
+### 执行离开候选人的请求
```
Comment (eshaben) [inline node-operators/networks/collator.md:168]: this is missing header attribute, should be `{: #execute-request-to-leave-candidates }` still

File: node-operators/networks/collator.md [hunk 7]
```diff
@@ -174,138 +177,127 @@ console.log(`Candidate pool size is: ${candidatePool.length}`);
...
- 6. 选择目标候选人账户（在提交`scheduleLeaveCandidates`函数后，任何人都可以在延迟时段过后执行请求）
+ 6. 选择目标候选人账户（在提交`scheduleLeaveCandidates` extrinsic后，任何人都可以在退出生效期过后执行请求）
 
- 7. 提交交易。根据向导指引使用创建账户时的密码进行交易签名
+  7. 提交交易。根据向导指引使用创建账户时的密码进行交易签名
 
 ![Execute leave candidates request](/images/node-operators/networks/collators/collator-polkadotjs-10.png)
 
-#### 取消离开候选人的请求 {: #cancel-request-to-leave-candidates }
+### 取消离开候选人的请求
```
Comment (eshaben) [inline node-operators/networks/collator.md:188]: this should have this attribute still `{: #cancel-request-to-leave-candidates }`

File: node-operators/networks/collator.md [hunk 8]
```diff
@@ -174,138 +177,127 @@ console.log(`Candidate pool size is: ${candidatePool.length}`);
...
-!!! 注意事项
-    Moonriver收集人的绑定数量在早期网络启动阶段为100枚MOVR。作为收集人，如果您所绑定的数量等于或少于{{ networks.moonriver.staking.min_can_stk }}枚MOVR，您将无法减少您的绑定数量。
-
-想要减少绑定数量，您需要先发起请求并等待延迟时段，随后执行请求。只要请求还未被执行，您仍可随时[取消请求](#cancel-request)。
-
-在Moonbase Alpha上会有{{ networks.moonbase.delegator_timings.revoke_delegations.rounds }}轮（{{ networks.moonbase.delegator_timings.revoke_delegations.hours }}小时）的延迟，在Moonriver上会有{{ networks.moonriver.delegator_timings.revoke_delegations.rounds }}轮（{{ networks.moonriver.delegator_timings.revoke_delegations.hours }}小时）的延迟。
+想要减少绑定数量，您需要先发起请求并等待[退出生效期](#collator-timings)，随后执行请求。只要请求还未被执行，您仍可随时[取消请求](#cancel-request)。
 
-#### 发起请求 {: #schedule-request }
+#### 发起减少绑定数量的请求
```
Comment (eshaben) [inline node-operators/networks/collator.md:241]: should be `{: #schedule-bond-less-request }`

File: node-operators/networks/collator.md [hunk 9]
```diff
@@ -174,138 +177,127 @@ console.log(`Candidate pool size is: ${candidatePool.length}`);
...
- 7. 提交交易。根据向导指引使用创建账户时的密码进行交易签名
+   7. 提交交易。根据向导指引使用创建账户时的密码进行交易签名
 
 ![Schedule Candidate Bond Less](/images/node-operators/networks/collators/collator-polkadotjs-12.png)
 
-当交易确认，您将需要等待延迟时段，然后才能为您执行减少绑定数量的请求。如果您尝试在延迟时段前操作，该操作将会失败，并且您将看到来自Polkadot.js Apps的`parachainStaking.PendingDelegationRequest`错误。
+当交易确认，您将需要等待退出生效期，然后才能为您执行减少绑定数量的请求。如果您尝试在退出生效期前操作，该extrinsic将会失败，并且您将看到来自Polkadot.js Apps的`parachainStaking.PendingDelegationRequest`错误。
 
-#### 执行请求 {: #execute-request }
+#### 执行减少绑定数量的请求
```
Comment (eshaben) [inline node-operators/networks/collator.md:262]: `{: #execute-bond-less-request }`

File: node-operators/networks/collator.md [hunk 10]
```diff
@@ -174,138 +177,127 @@ console.log(`Candidate pool size is: ${candidatePool.length}`);
...
 
- 7. 提交交易。根据向导指引使用创建账户时的密码进行交易签名
+   7. 提交交易。根据向导指引使用创建账户时的密码进行交易签名
 
 ![Execute Candidate Bond Less](/images/node-operators/networks/collators/collator-polkadotjs-13.png)
 
 交易确认后，您可以在**Accounts**标签处查看您的可用余额。请注意，请求已执行，您的余额已更新。
 
-#### 取消请求 {: #cancel-request }
+#### 取消减少绑定数量的请求
```
Comment (eshaben) [inline node-operators/networks/collator.md:283]: `{: #cancel-bond-less-request }`

File: snippets/text/staking/staking-definitions.md [hunk 11]
```diff
@@ -1,10 +1,15 @@
...
- - **委托人候选人限额（Maximum candidates per delegator）** — 每个委托人能提名的最高候选人数量
- - **轮次（Round）** — 执行质押操作的时效衡量单位。例如，在下一轮开始时制定新的委托。当绑定减少或撤销授权时，资金会在体格轮次后返还。
\ No newline at end of file
+ - **轮次（Round）**—— 质押行动被执行所需的固定区块数量。举例而言，新的委托将会在下个轮次开始时执行。当减少绑定数量或是撤销委托时，资金将会在一定轮次后退回
+ - **候选收集人（Candidates）**—— 在获得足够质押量后进入收集人有效集，才有资格产生区块的节点运营商
+ - **收集人（Collators）**—— 获选成为区块生产者的候选收集人。他们从收集用户的交易记录并为中继链提供状态转换证明以供验证
+ - **委托人（Delegators）**—— 质押Token的Token持有者，为特定的候选收集人担保。任何持有超过最低数量且能够[自由支配](https://wiki.polkadot.network/docs/learn-accounts#balance-types)的Token的人皆能够成为委托人
+ - **最低委托持有量（Minimum delegation per candidate）**——委托人要委托候选收集人所需的最低Token数量
+ - **候选收集人的委托人限额（Maximum delegators per candidate）**——每个候选收集人能接受的最高可获得奖励的委托人数量（根据质押数量决定）
+ - **最高委托量（Maximu delegations）**—— 委托人能够委托的最高候选收集人数量
```
Comment (eshaben) [inline snippets/text/staking/staking-definitions.md:9]: missing an m at the end of Maximum

**Review by eshaben — CHANGES_REQUESTED** [review]

**Comment by hyd628** [comment]
@eshaben Thanks! all feedback has been incorporated.

**Review by eshaben — APPROVED** [review]
Awesome! LGTM ✨🚀

**Pull request merged**